modules/rosa-hcp-creating-account-wide-sts-roles-and-policies.adoc

```diff
@@ -11,11 +11,11 @@ endif::[]
 [id="rosa-sts-creating-account-wide-sts-roles-and-policies_{context}"]
 = Creating the account-wide STS roles and policies
 
-Before using the {product-title} (ROSA) CLI (`rosa`) to create {hcp-title-first} clusters, create the required account-wide roles and policies, including the Operator policies.
+Before you create your {hcp-title-first} cluster, you must create the required account-wide roles and policies.
 
 [NOTE]
 ====
-{hcp-title} clusters require account and Operator roles with AWS managed policies attached. Customer managed policies are not supported. For more information regarding AWS managed policies for {hcp-title} clusters, see link:https://docs.aws.amazon.com/ROSA/latest/userguide/security-iam-awsmanpol-account-policies.html[AWS managed policies for ROSA account roles].
+Specific AWS managed policies for {hcp-title} must be attached to each role. Customer managed policies must not be used with these required account roles. For more information regarding AWS managed policies for {hcp-title} clusters, see link:https://docs.aws.amazon.com/ROSA/latest/userguide/security-iam-awsmanpol-account-policies.html[AWS managed policies for ROSA].
```

Comment:
AWS-managed/AWS managed and customer-managed/customer managed are inconsistent in the docs. 
```suggestion
Specific AWS-managed policies for {hcp-title} must be attached to each role. Customer-managed policies must not be used with these required account roles. For more information regarding AWS managed policies for {hcp-title} clusters, see link:https://docs.aws.amazon.com/ROSA/latest/userguide/security-iam-awsmanpol-account-policies.html[AWS managed policies for ROSA].
```